.travis/linux_travis_arm64_setup.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+# Basic tools
+
+set -x
+
+if [ ! -z "${DOCKERHUB_PULL_USERNAME:-}" ]; then
+  set +x
+  echo "${DOCKERHUB_PULL_PASSWORD}" | docker login --username "${DOCKERHUB_PULL_USERNAME}" --password-stdin
+  set -x
+fi
+
+sudo apt-get update -qq
+sudo apt-get install -qq mysql-client zip jq expect nfs-kernel-server build-essential curl git libnss3-tools libcurl4-gnutls-dev docker-compose
+
+# Copy docker-compose to /usr/local/bin because Travis' pre-installed version leads to exec format error
+sudo cp /usr/bin/docker-compose /usr/local/bin/docker-compose
+
+curl -sSL --fail -o /tmp/ngrok.zip https://bin.equinox.io/c/4VmDzA7iaHb/ngrok-stable-linux-arm64.zip && sudo unzip -o -d /usr/local/bin /tmp/ngrok.zip
+
+# Without this .curlrc CircleCI linux image doesn't respect mkcert certs
+echo "capath=/etc/ssl/certs/" >>~/.curlrc
+
+. ~/.bashrc
+
+npm install -g bats
```

Comment:
I didn't know you could do it this way. Easier than most of the ways I've done it. Oh, but I wonder if this gets bats-core/bats-core or the old unmaintained one. (Edit: I checked; it gets bats 1.2.1, which is from bats-core)